bool stop_current 
geometry_msgs/PoseStamped goal_location
string location_name 
